mappings/net/minecraft/item/SuspiciousStewItem.mapping

```diff
@@ -2,6 +2,10 @@ CLASS net/minecraft/unmapped/C_kkuffgzc net/minecraft/item/SuspiciousStewItem
 	FIELD f_lrwjllbv EFFECT_ID_KEY Ljava/lang/String;
 	FIELD f_ssywxyld EFFECT_DURATION_KEY Ljava/lang/String;
 	FIELD f_wopcliea EFFECTS_KEY Ljava/lang/String;
+	METHOD m_cryeecjp consumeStatusEffects (Lnet/minecraft/unmapped/C_sddaxwyk;Ljava/util/function/Consumer;)V
+		ARG 0 stack
+		ARG 1 consumer
+			COMMENT handles the usage of each new status effect instance after it's created
```

Comment:
oh, tiny nitpick;
```suggestion
			COMMENT handles the usage of each new status effect instance after it is created
```
great usage of javadocs in order to prevent a `getStatusEffectsButPutItOnAList`! :P